src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemSectionsPageViewModelTest.kt

```diff
@@ -28,7 +28,7 @@ class SoftwareSystemSectionsPageViewModelTest : ViewModelTest() {
         assertThat(viewModel.sectionsTable)
```

Comment:
Would you mind changing this assert to be like this one? https://github.com/avisi-cloud/structurizr-site-generatr/blob/6f1a6a96abeb75d9ace637989ebb077e7ea2b4ba/src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemContainerSectionsPageViewModelTest.kt#L31-L41 ? The current assert was already slightly weird. slightly

Reply:
@jp7677 added! Thanks for suggestion. Looks much cleaner.